components/DelegateCard.tsx

```diff
@@ -0,0 +1,166 @@
+import useRealm from '../hooks/useRealm'
+import useWalletStore from '../stores/useWalletStore'
+import React, { useState, FunctionComponent } from 'react'
+import {
+  Transaction,
+  TransactionInstruction,
+  Keypair,
+  PublicKey,
+} from '@solana/web3.js'
+import Input from './inputs/Input'
+import {
+  withSetGovernanceDelegate,
+  getGovernanceProgramVersion,
+} from '@solana/spl-governance'
+import { sendTransaction } from 'utils/send'
+import { CashIcon, CreditCardIcon } from '@heroicons/react/solid'
+import Button from './Button'
+import Checkbox from '@components/inputs/Checkbox'
+
+const DelegateCard = () => {
+  const {
+    realm,
+    ownTokenRecord, // community
+    ownCouncilTokenRecord, // council
+  } = useRealm()
+  const [isLoading, setLoading] = useState<boolean>(false)
+  const wallet = useWalletStore((s) => s.current)
+  const connection = useWalletStore((s) => s.connection.current)
+  const [delegateKey, setDelegateKey] = useState('')
+  const [tokenType, setTokenType] = useState<'community' | 'council'>('council')
+
+  const handleDelegate = async () => {
+    const signers: Keypair[] = []
+    const instructions: TransactionInstruction[] = []
+    setLoading(true)
+
+    if (!realm || !realm?.account?.config?.councilMint || !wallet?.publicKey) {
+      return
+    }
+
+    try {
+      const programVersion = await getGovernanceProgramVersion(
+        connection,
+        realm.owner // governance program public key
+      )
+
+      const tokenMintToDelegateWith =
+        tokenType === 'council'
+          ? realm?.account?.config?.councilMint
+          : realm.account.communityMint
+
+      withSetGovernanceDelegate(
+        instructions,
+        realm.owner, // publicKey of program/programId
+        programVersion, // program version of realm
+        realm.pubkey, // realm public key
+        tokenMintToDelegateWith, // mint of governance token
+        wallet?.publicKey, // governingTokenOwner (walletId) publicKey of for tokenOwnerRecord of this wallet
+        wallet?.publicKey, // governanceAuthority: publicKey of connected wallet
+        new PublicKey(delegateKey) // public key of wallet who to delegated vote to
+      )
+      const recentBlockhash = await connection.getRecentBlockhash()
+      const transaction = new Transaction({
+        recentBlockhash: recentBlockhash.blockhash,
+      })
```

Comment:
nit: I think the is not needed because `sendTransaction` sets the `blockhash` and `feePayer`

Reply:
replaced this with 
```      
const transaction = new Transaction()
transaction.add(...instructions)
```